tests/framework/cli/test_cli.py

```diff
@@ -371,22 +363,23 @@ def test_project_commands_valid_clipy(self, mocker, fake_metadata):
             return_value=Module(cli=cli),
         )
         kedro_cli = KedroCLI(fake_metadata.project_path)
-        assert len(kedro_cli.project_groups) == 7
+        assert len(kedro_cli.project_groups) == 2
         assert kedro_cli.project_groups == [
-            catalog_cli,
-            jupyter_cli,
-            pipeline_cli,
-            micropkg_cli,
-            project_group,
-            registry_cli,
+            # catalog_cli,
+            # jupyter_cli,
+            # pipeline_cli,
+            # micropkg_cli,
+            # project_group,
+            # registry_cli,
```

Comment:
Do you want to keep these comments or is that still a left-over clean up?

Reply:
cleanup, removed!